packages/@ember/polyfills/index.ts

```diff
@@ -5,4 +5,5 @@ let merge = MERGE ? deprecatedMerge : undefined;
 
 // Export `assignPolyfill` for testing
 export { default as assign, assign as assignPolyfill } from './lib/assign';
+export { default as WeakSet } from './lib/weak_set';
```

Comment:
The `@ember/polyfills` package does _not_ intend to have this as public API, can you export it as an underscored property?

Reply:
done